ruby-build-and-test.yml

```diff
@@ -92,9 +92,9 @@ status_pending:
     - export pipeline_status="pending"
     - *status_report
 
-intel_19_1_2_gcc_8_5_0:
+intel_19_1_2_gcc_10_3_1:
   variables:
-    SPEC: "${PROJECT_RUBY_VARIANTS} %intel@19.1.2.gcc.8.5.0 ${PROJECT_RUBY_DEPS}"
+    SPEC: "${PROJECT_RUBY_VARIANTS} %intel@19.1.2.gcc.10.3.1 ${PROJECT_RUBY_DEPS}"
```

Comment:
In Umpire Gcc @ 10.3.1 won't work with intel @ 19.1

Reply:
I'll have to test this. I had quite clear results for my experiment using the system gcc (assuming it was 8.5.0) vs 10.3.1.